cdk.tf/vercel.json

```
@@ -190,6 +190,10 @@
     {
       "source": "/crash-reporting",
       "destination": "https://www.terraform.io/cdktf/telemetry#crash-reporting"
+    },
+    {
```

Can we add more "this is not for leisure but depended on" context or does that break the build?
```suggestion
    { "//": "This is used by the cdktf provider subcommands"
```

---

Yeah, such a comment seems reasonable – let's see whether Vercel allows for the `//` attribute.